src/exchange/wizard-hosted-creation/add-domain-name/add-domain-name.html

```diff
@@ -0,0 +1,231 @@
+<h1 data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_header')"></h1>
+<hr>
+
+<div data-ng-if="$ctrl.loaders.isInitialRetrievalRunning" class="loader text-center"></div>
+
+<div class="mt-5" data-ng-if="!$ctrl.loaders.isInitialRetrievalRunning">
+    <p>
+        <span data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_ifNoDomain')"></span>
+        <a data-ng-href="{{:: $ctrl.urlToOrderDomains }}"
+           target="_blank"
+           title="{{:: $ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_goToOffers') }}">
+            <span data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_goToOffers')"></span>
+            <span class="fa fa-external-link" aria-hidden="true"></span>
+        </a>
+    </p>
+
+    <label data-ng-bind="$ctrl.translator.tr('exchange_wizardHostedCreation_addDomainName_domainNameSelection_header')" for="domainNameSelection"></label>
```

Comment:
Any reason to not use this template :

Reply:
Actually yes, I'm following the mock-up provided by CX and the `label` is supposed to be above the input field and we are inside a `form-inline` to have the `select` at the same level as the `checkbox`.

As `label` doesn't have to be part of a `form` then I chose to have it above.